components/ui/Svg.tsx

```diff
@@ -63,6 +63,8 @@ export default function Svg(props: SvgProps) {
 			});
 	}, [srcUrl]);
 
+	useEffect(() => () => setSvgElement(null), []);
+
```

Comment:
@LemToUp could we combine this `useEffect` hook with the one above? That will also solve the case if `srcUrl` changes I guess.